configure.ac

```diff
@@ -110,7 +111,16 @@ case "$host" in
         LT_EXTRA="-os2dllname SDL2img"
         ;;
 esac
-AM_CONDITIONAL(USE_IMAGEIO, test x$enable_imageio = xyes)
+case "$host" in
+    *-*-mingw*)
+        AC_SUBST([MINGW],1)
+        ;;
+    *)
+        AC_SUBST([MINGW],0)
+        ;;
+esac
+AM_CONDITIONAL(USE_IMAGEIO, $enable_imageio)
```

Comment:
This needs to be the old conditional:
`AM_CONDITIONAL(USE_IMAGEIO, test x$enable_imageio = xyes)`